README.md

```diff
@@ -18,6 +18,26 @@ If you are considering making a PR, please take a look at the Waffle board to se
 
 The following steps walk through getting the application running. For contributing guidelines see [here](https://github.com/codebar/planner/blob/master/CONTRIBUTING.md).
 
+## Getting Started Through Docker
+
+You will need to have Docker installed, https://docker.com/
+
+The current Dockerfile and docker-compose were closely copied from the guide: https://docs.docker.com/compose/rails/
+
+1. Clone the project
+
+Run this to build a docker environemnt:
```

Comment:
Thanks for testing richard, apart from these changes we will need to change the travisci and other connected tools to adapt to these new database.yml changes, not sure who has access to all of those.

Will change the layout as soon as i can